CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ These are changes that will probably be included in the next release.
 ### Removed
 ### Fixed
 
+## [v0.4.24] - 2021-08-24
```

Comment:
0.4.23 tag does exist, but has no history in the changelog, therefore we go to 0.4.24